package.json

```diff
@@ -142,6 +142,7 @@
     "@polkadot/util": "^13.4.3",
     "@polkadot/util-crypto": "^13.4.3",
     "@projectlibertylabs/graph-sdk": "^2.0.1",
+    "@projectlibertylabs/siwf": "^2.2.1",
```

Comment:
Unnecessary? Dupe of `siwfv2` below. When we eventually get rid of v1/v2 for SIWF, we can collapse to this, but for now I think this is unnecessary.

```suggestion
```